src/flows/components/header/Submit.tsx

```diff
@@ -151,9 +65,9 @@ export const Submit: FC = () => {
           text={runMode}
           className="flows--submit-button"
           icon={IconFont.Play}
-          submitButtonDisabled={!hasQueries}
+          submitButtonDisabled={hasQueries === false}
```

Comment:
this change isn't needed right? just the filter above to make the button disabled?

Reply:
It isn't necessary, but i think it makes it a bit more understandable